azure-pipelines.yml

```diff
@@ -176,15 +176,15 @@ jobs:
     displayName: 'Test the wheels with pytest'
 
   - script: |
-      pip install openml pandas
+      pip install -e ".[examples]"
       pip install "papermill==1.2.1"
       cd examples
       for n in *.ipynb
       do
         papermill --start_timeout 2000 $n -
       done
     failOnStderr: true
-    displayName: 'Test the wheels on jupyter notebooks with papermill'
+    displayName: 'Test jupyter notebooks with papermill'
```

Comment:
I count 2 changes of this particular phrase so far, but shouldn't there be another one?

Reply:
Good catch! FIxed :)